arches_component_lab/src/arches_component_lab/cards/utils.ts

```diff
@@ -0,0 +1,37 @@
+export function extractFileEntriesFromPayload(
+    payloadObject: Record<string, unknown>,
+) {
+    const fileEntries: { file: File; nodeId: string }[] = [];
+
+    function recurse(currentObject: Record<string, unknown>): void {
+        for (const key in currentObject) {
+            if (!Object.prototype.hasOwnProperty.call(currentObject, key)) {
+                continue;
+            }
```

Comment:
Did you consider just checking for the existence of the `aliased_data`? Instead it looks like this will descend into `provisionaledits` and interpret nested objects there which I doubt we want. Should also get a small performance boost.

```
for const [key, val] of Object.entries() {
   if (val instanceof File) { ... }
   if (val.aliased_data) {
      recurse() 
}
```